modules/default/calendar/calendar.js

```diff
@@ -114,12 +116,12 @@ Module.register("calendar", {
 
 			this.addCalendar(calendar.url, calendar.auth, calendarConfig);
 
-			// Trigger ADD_CALENDAR every fetchInterval to make sure there is always a calendar
+			/*			// Trigger ADD_CALENDAR every fetchInterval to make sure there is always a calendar
```

Comment:
Why not remove the code if it isnt used here anymore?

Reply:
will fix